### DISCUSSION TITLE
Avalonia UI Reveal

### DISCUSSION THREAD

**Discussion by JordanMarr** [fsprojects/Avalonia.FuncUI#268] [General]
What is it? I know someone around here knows.

![image](https://user-images.githubusercontent.com/1030435/217962431-24159b9e-8e04-46f8-a9ae-1e5a02258b2a.png)

**Comment by AngelMunoz** [comment]
I'm thrilled to know what that is, hopefully stable 1.0 release of avalonia (including mobile/wasm stuff)

**Reply by Numpsy** [reply]
https://avaloniaui.net/xpf

though maybe there will be a stable Avalonia release to base that on
